lib/Doctrine/CouchDB/HTTP/StreamClient.php

```diff
@@ -38,21 +38,31 @@ class StreamClient extends AbstractHTTPClient
      * Perform a request to the server and return the result converted into a
      * Response object. If you do not expect a JSON structure, which
      * could be converted in such a response object, set the forth parameter to
-     * true, and you get a response object retuerned, containing the raw body.
+     * true, and you get a response object returned, containing the raw body.
      *
      * @param string $method
      * @param string $path
      * @param string $data
      * @param bool $raw
+     * @param array $headers
      * @return Response
      * @throws HTTPException
      */
-    public function request( $method, $path, $data = null, $raw = false )
+    public function request( $method, $path, $data = null, $raw = false, array $headers = array())
```

Comment:
yes please add it, you cannot extend an interface this way, it is not proper :-)